### DISCUSSION TITLE
When will new projects would be added to GitHub Octernship?

### DISCUSSION THREAD

**Discussion by rythmn1111** [community/community#76875] [GitHub Education] [answered]
### Hi everyone,

Hi everyone, so I have a doubt regarding the octenship. So when i am viewing the page it is showing "The application window has closed for all current projects. New projects will be added soon." does this means that octenship has ended? I am confused about how does this works :(

**Comment by El-Hibry** [comment, marked as answer]
Hey,  GitHub Octernships program is currently in its off-season. The next application cycle will be open by January 2024. You can check the GitHub Octernships website for updates on the application timeline.

In the meantime, you can still prepare for the application by:

* Becoming a verified student on GitHub Global Campus.
* Becoming an active contributor on GitHub.
* Enrolling in an institution with work eligibility in one of the ten countries: India, Singapore, Indonesia, Malaysia, Vietnam, Philippines, Thailand, Mexico, Nigeria, and Colombia.
* Learning about open source software development.
* Practicing your coding skills.

**Reply by A-nn-e** [reply]
What do you mean by being an active contributor? How to know if I am an active contributor or not?

**Reply by Abiji-2020** [reply]
And to the question of active contributor and how to Learn or contribute to Open Source Software Development?

**Reply by iforgotn** [reply]
Why is it not in the us

**Reply by rythmn1111** [reply]
when you contribute to some repo it shows on your profile that's how

**Reply by JeyGR** [reply]
Is it like, every January is the start of the program?

**Comment by ojuschugh1** [comment]
@El-Hibry Hey what is the GitHub octernship website link? Could you please share it as I cannot access it on education.github.com?

**Reply by dvd-22** [reply]
I know it is a bit late answer, but it seems that either Octernships are discontinued or they changed something on the requirements

**Reply by VatsShaurya** [reply]
Hey, Is octernship discontinued? I could not find the application link. Did you get the link?

**Reply by ananthramj** [reply]
I have the same question. While I've pretty much only made an account on GitHub and done nothing thus far, I was surprised when the link to it in an article I stumbled upon didn't work. Have Octernships been discontinued?

**Comment by ritankarsaha** [comment]
Same for me. Are Github Octernships discontinued ?
